### PR TITLE
Move DOCHOSTUIDBLCLK and DOCHOSTUIFLAG enums to separate files

### PR DESCRIPTION
**Pull request by gpetrou**
##  Proposed changes

- Create files for DOCHOSTUIDBLCLK and DOCHOSTUIFLAG enums.
- Remove corresponding enums from NativeMethods.cs and replace their usages with the above enums.
- Update DOCHOSTUIINFO definition.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2242)

### CODE REVIEW

File: src/Common/src/Interop/Mshtml/Interop.DOCHOSTUIFLAG.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal static partial class Interop
+{
+    internal static partial class Mshtml
+    {
+        public enum DOCHOSTUIFLAG
```
Comment (hughbe) [inline src/Common/src/Interop/Mshtml/Interop.DOCHOSTUIFLAG.cs]: Flags

File: src/System.Windows.Forms/src/System/Windows/Forms/WebBrowser.cs [hunk 2]
```diff
@@ -1505,26 +1505,26 @@ int UnsafeNativeMethods.IDocHostUIHandler.GetHostInfo(NativeMethods.DOCHOSTUIINF
             {
                 WebBrowser wb = (WebBrowser)Host;
 
-                info.dwDoubleClick = (int)NativeMethods.DOCHOSTUIDBLCLICK.DEFAULT;
-                info.dwFlags = (int)NativeMethods.DOCHOSTUIFLAG.NO3DOUTERBORDER |
-                                (int)NativeMethods.DOCHOSTUIFLAG.DISABLE_SCRIPT_INACTIVE;
+                info.dwDoubleClick = (int)DOCHOSTUIDBLCLK.DEFAULT;
+                info.dwFlags = (int)DOCHOSTUIFLAG.NO3DOUTERBORDER |
```
Comment (hughbe) [inline src/System.Windows.Forms/src/System/Windows/Forms/WebBrowser.cs]: Can we make these fields the appropriate Enum?

File: src/Common/src/Interop/Mshtml/Interop.DOCHOSTUIFLAG.cs (hunk 1 above)
Comment (gpetrou) [reply src/Common/src/Interop/Mshtml/Interop.DOCHOSTUIFLAG.cs]: Done.

File: src/System.Windows.Forms/src/System/Windows/Forms/WebBrowser.cs (hunk 2 above)
Comment (gpetrou) [reply src/System.Windows.Forms/src/System/Windows/Forms/WebBrowser.cs]: Sure.

**Comment by RussKie** [comment]
@msftbot merge if @hughbe approves

**Pull request merged**